Automattic/wp-calypso
# Full importer: turn on the feature on staging and production environments

#### Changes proposed in this Pull Request

Full importer: turn on the feature on staging and production environments 🚀 

#### Testing instructions

* Production env
* Go to `/start/importer/capture?siteSlug={SLUG}`
* Check if the capture page is rendered



Closes #57131
